fire/core.py

```diff
@@ -166,13 +171,18 @@ class FireExit(SystemExit):
 
   Contains the trace of the Fire program.
 
-  If not caught, then a FireExit will cause the program to exit with a non-zero
-  status code.
+  Note:
+    Since this exception inherits from SystemExit/BaseException, you'll have to
+    explicitly catch it with `except SystemExit` or `except FireExit`.
   """
-
-  def __init__(self, status, data):
-    super(FireExit, self).__init__(status)
-    self.trace = data
+  def __init__(self, code, trace):
```

Comment:
https://github.com/google/apitools/blob/master/default.pylintrc - wasn't actually that useful as a pylintrc, but it was good enough so I could set indent to '  ' and a line length and fix the lint errors for you :P